modules/aws/ec2/README.md

```diff
@@ -82,15 +116,8 @@ sudo systemctl enable docker.service
 sudo systemctl start docker.service
 
 sudo cat << EOL > /usr/.env
-DATABASE_NAME=${var.database_name}
-DATABASE_HOST=${module.rds_cluster_setup.rds_serverless_endpoint}
-DATABASE_PORT=${module.rds_cluster_setup.rds_serverless_port}
-DATABASE_USER=${module.rds_cluster_setup.rds_serverless_master_username}
-DATABASE_PASS=${module.rds_cluster_setup.rds_serverless_master_password}
```

Comment:
all these values need to be included if you are creating a django application on the ec2 server. That is why they were in this example.

Reply:
I think if you provide an example but don't provide everything you would need to get it working and assume that you'd need another module that doesn't make a great example and could potentially lead to confusion.

Happy to add it back in but then the example is dependent on a RDS Cluster which is massive overkill as a getting started example with the module.